scripts/eslint.sh

```diff
@@ -1,20 +0,0 @@
-#!/bin/bash
-
-set -e
-
-mkdir -p tmp
-
-npm run lint | tee tmp/lint-output
-
-problems=`cat tmp/lint-output | grep -oE '[0-9]* problems' | grep -o "[0-9]*"`
-ratchet="100"
```

Comment:
Not sure if correct but I thought that this is the value Travis uses to mark builds as failed when there are too many linting issues (in this case more than 100).